src/StateConfig.php

```diff
@@ -80,6 +80,25 @@ public function allowTransitions(array $transitions): StateConfig
         return $this;
     }
 
+    /**
+    * @param \Illuminate\Database\Eloquent\Model $model
+    * @param string $fromClass
+    */
+    public function allowedTransitionsFrom(Model $model, string $fromClass)
```

Comment:
I'm not sure this is the right name. You're not returning a list of transitions, but rather a list of states the current state can transition to. 

Maybe `transitionableStates`? Maybe you can come up with a better name?